### DISCUSSION TITLE
Had to change the desktop entry on my Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by ve7kfx** [markqvist/Reticulum#622] [Sideband]
`[Desktop Entry]
Comment[en_US]=Messaging, telemetry and remote control over LXMF
Comment=Messaging, telemetry and remote control over LXMF
Encoding=UTF-8
Exec=/home/cyberdeck/.local/bin/sideband
GenericName[en_US]=LXMF client
GenericName=LXMF client
Icon=/home/cyberdeck/.local/share/icons/hicolor/512x512/apps/io.unsigned.sideband.png
Categories=Utility
MimeType=
Name[en_US]=Sideband
Name=Sideband
Path=/home/cyberdeck/.local/bin/
StartupNotify=true
Terminal=false
TerminalOptions=
Type=Application
Version=1.0
X-KDE-SubstituteUID=false
X-KDE-Username=
`

**Comment by markqvist** [comment]
Interesting to know, thanks for the info. The code block above ended up a bit jumbled, could you describe what parts you had to change? That'd make it easy for me to incorporate it into the setup script itself so it works for everyone.

**Reply by ve7kfx** [reply]
Sorry about that, I changed the paths to the icons and the binary, here is the code block with better formatting:
```shell
[Desktop Entry]
Comment[en_US]=Messaging, telemetry and remote control over LXMF
Comment=Messaging, telemetry and remote control over LXMF
Encoding=UTF-8
Exec=/home/cyberdeck/.local/bin/sideband
GenericName[en_US]=LXMF client
GenericName=LXMF client
Icon=/home/cyberdeck/.local/share/icons/hicolor/512x512/apps/io.unsigned.sideband.png
Categories=Utility
MimeType=
Name[en_US]=Sideband
Name=Sideband
Path=/home/cyberdeck/.local/bin/
StartupNotify=true
Terminal=false
TerminalOptions=
Type=Application
Version=1.0
X-KDE-SubstituteUID=false
X-KDE-Username=
```

**Reply by markqvist** [reply]
Thanks. Was this installed via `pip` or `pipx`?

**Reply by ve7kfx** [reply]
with pip
